orchprovider/k8s/v1/k8s.go

```diff
@@ -864,15 +864,23 @@ func (k *k8sOrchestrator) createControllerDeployment(volProProfile volProfile.Vo
 	glog.Infof("Adding controller for volume 'name: %s'", vsm)
 	var tolerationSeconds int64 = 0
 
+	ctrlLabelSpec := map[string]string {
+		string(v1.VSMSelectorKey):               vsm,
+		string(v1.PVCSelectorKey):               pvc,
+		string(v1.VolumeProvisionerSelectorKey): string(v1.JivaVolumeProvisionerSelectorValue),
+		string(v1.ControllerSelectorKey):           string(v1.JivaControllerSelectorValue),
+	}
+
+	//Add the application label to the replica deployment if it exists.
```

Comment:
The comment mentions replica . is it replica?

Reply:
cool. done!